### ISSUE TITLE
Heat exchanger example invalid

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#741]
**Reported by msielemann on 10 Jun 2009 12:36 UTC**
In the example of a heat exchanger, the parameter use_HeatTransfer is set to false for both dynamic pipes. Therefore, no heat is actually transferred from one place to the other. Setting those parameters to true in the following class fixes the problem.

```
Modelica_Fluid.Examples.HeatExchanger.BaseClasses.BasicHX
```

Furthermore, the model 
```
Modelica_Fluid.Examples.HeatExchanger.HeatExchangerSimulation
```
prescribes mass flow rates. Therefore, the corresponding parameters from_dp should be set to false (in order to avoid a dozen of small and unnecessary nonlinear equation systems).

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/741

**Comment by modelica-trac-importer** [comment]
**Comment by fcasella on 9 May 2012 14:48 UTC**
moved from https://trac.modelica.org/Modelica_Fluid/ticket/94

**Comment by modelica-trac-importer** [comment]
**Modified by fcasella on 9 May 2012 14:56 UTC**

**Comment by modelica-trac-importer** [comment]
**Modified by rfranke on 27 Dec 2012 12:06 UTC**

**Comment by modelica-trac-importer** [comment]
**Changelog removed by rfranke on 27 Dec 2012 12:06 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by rfranke on 27 Dec 2012 12:09 UTC**
The setting use_heatTransfer has been fixed to true in c43eeb40e330b6d9bbff374ea0c272420de2cdd4. Moreover ConstantPropertyLiquidWater and ConstantHeatTransfer have been replaced with the more interesting StandardWater and LocalPipeFlowHeatTransfer, respectively -- from_dp has not been changed as the small equation systems disappeared with StandardWater.
